playbooks/roles/tower/defaults/main.yml

```diff
@@ -29,6 +29,11 @@ tower_credential_bundle_github_name: "Tower Default Github Credentials"
 tower_credential_bundle_github_desc: "Default Tower Credential Bundle for Github"
 tower_credential_bundle_github_type: 'scm'
 
+aws_credential_list:
```

Comment:
Could this be set to:
```
aws_credential_list: []
```

Reply:
If we run the plays without passing in the private repo, it'll create a blank.AWS credential etc, but sure we can set it to []